platform/engineering/code_review_guidelines.md

```diff
@@ -4,41 +4,58 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
-      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
-    * After having your pull request reviewed by your team you can click the *Ready for review* button in the merge section of your pull request.
-      * An engineer from the VSP team will aim to review within 24 hours (business days) after the pull request has been reviewed by your team and you have been marked it *Ready for review*. Each additional round of changes will be reviewed within 24 hours.  If you need a review faster, escalate via your DSVA product owner.
-      * **Do not post in Slack channels requesting VSP code reviews** unless it has been more than 24 hours since requesting review on GitHub.
-    * After sending your code for review, don't add additional changes! Code needs to be stable for a safe review.
-    * If non-trivial changes to your code are made *after approval by VSP team* and submitted without a re-review, your change is subject to being reverted.
-
-* **First review**: 
-  * Use a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
-  * Tag your PR with your team-specific label(s)
-  * For this first review, your code should be reviewed and approved by a member of your team. 
-  * First review should cover:
-    * **Correctness** of the code (does it correctly implement the described feature?)
-    * **Code quality**: is it readable? Is it language-idiomatic?
-    * **Does it have screenshots of the change?** (FE changes)
-
-* **Description**: make it clear what the change is describing.
-    * **PR title** should be a concise summary of the change.
-    * In your **PR description**, discuss how code changes are related to the pull request objectives.
-    * Reference the **GitHub issues** your PR relates to, in order to provide reviewers more context.
-
-* **Size/complexity**: keep your changes scoped to one feature at a time. This means:
-    * **Size**: changes should be smaller than **300** changed lines (not including tests or whitespace). *If your change can't be smaller than this, talk to the team in #vfs-platform-support before asking for a review.*
+We expect that most developers have worked with GitHub and review processes before, but want to be clear about our norms to reduce confusion and help code flow through the process as quickly and reliably as possible.
+
+
+## The pull request review lifecycle in brief
+
+1. For initial review by your project team, create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+2. Once you are ready for the VSP team to review your PR, use the GitHub *Ready for review* button in the merge section of your pull request. An engineer should review the code within 2 days. 
```

Comment:
this seems like a policy change, not just a guidelines/wording update (1 day --> 2 days), was that intentional?

Reply:
1 business day like we discussed on slack